.github/workflows/publish.yml

```diff
@@ -12,9 +12,7 @@ jobs:
         with:
           submodules: true
       - name: Publish Site
-        # We use a forked action due to
-        # https://github.blog/2022-04-12-git-security-vulnerability-announced/ (https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-24765).
-        uses: kubewarden/hugo-gh-pages@fix-publishing
+        uses: chabad360/hugo-gh-pages@master
```

Comment:
Shouldn't we use a tagged version and leave the bots update the version for us?

Reply:
unfortunately this GH action doesn't have tags :(

I'm open to using another action